docs/src/content/home.ts

```diff
@@ -0,0 +1,262 @@
+import GitHubLogo from "../assets/socials/gh-logo";
+import TwitterLogo from "../assets/socials/tw-logo";
+import DiscordLogo from "../assets/socials/dc-logo";
+import VictorImage from "../assets/images/victor.png";
+import KyberImage from "../assets/images/trustedTeamsLogos/kyber.png";
+import JustimImage from "../assets/images/justin.jpg";
+import SynthetixImage from "../assets/images/trustedTeamsLogos/synthetix.png";
+import BrettImage from "../assets/images/brett.jpg";
+import AragoneOneImage from "../assets/images/trustedTeamsLogos/aragonOne.png";
+import RahulImage from "../assets/images/rahul.png";
+import ConnextImage from "../assets/images/trustedTeamsLogos/connext.png";
+import EstebanImage from "../assets/images/esteban.png";
+import DecentralandImage from "../assets/images/trustedTeamsLogos/decentraland.png";
+
+import { SocialsEnum } from "../components/ui/types";
+import { BANNER_LINK, SOCIALS_LINKS, Tools } from "../config";
+import VSCodeIcon from "../assets/tools/vs-code";
+import RunnerIcon from "../assets/tools/runner";
+import IgnitionIcon from "../assets/tools/ignition";
+import NetworkIcon from "../assets/tools/network";
+import SolidityImageMobile from "../assets/feature-cards/Mobile/SolidityImage.png";
+import SolidityImageDesktop from "../assets/feature-cards/Desktop/SolidityImage.png";
+import FlexibilityImageMobile from "../assets/feature-cards/Mobile/FlexibilityImage.png";
+import FlexibilityImageDesktop from "../assets/feature-cards/Desktop/FlexibilityImage.png";
+import ExtensibleImageMobile from "../assets/feature-cards/Mobile/ExtensibleImage.png";
+import ExtensibleImageDesktop from "../assets/feature-cards/Desktop/ExtensibleImage.png";
+import FastIterationImageMobile from "../assets/feature-cards/Mobile/FastIterationImage.png";
+import FastIterationImageDesktop from "../assets/feature-cards/Desktop/FastIterationImage.png";
+
+const defaultSocialsItems = [
```

Comment:
social items are common for the entire website - so please move them to config file